panther_cli/main.py

```diff
@@ -15,15 +15,16 @@
 '''
 
 import argparse
-from collections import defaultdict
+# from collections import defaultdict
```

Comment:
Are you leaving this commented for a reason?

Reply:
Nope, good catch. Removed.